azure-pipelines.yml

```diff
@@ -39,7 +37,12 @@ steps:
       cd terraform
       terraform init
       terraform plan -out=tfplan -input=false
-      terraform apply "tfplan"
+      if [$TF_VAR_BRANCH -eq "master"]
+      then
+        terraform apply "tfplan"
+      elif [$FORCE -eq "True"]
+        terraform apply "tfplan"
+      fi
```

Comment:
Is thais change supposed to be in here?

Reply:
yeah I wanted to try something, so we can use the build on a PR and see what it might do.